Fix sphinx-doc target: it already runs in the 'doc' directory, setting
WORKING_DIRECTORY to 'doc' is wrong.

--- cmake/sphinx.cmake~	2011-01-14 17:04:28.000000000 +0100
+++ cmake/sphinx.cmake	2011-11-01 17:14:52.000000000 +0100
@@ -41,14 +41,12 @@
     ADD_CUSTOM_TARGET(sphinx-doc ALL
       COMMAND ${SPHINX_BUILD} -b html ${CMAKE_CURRENT_BINARY_DIR}/sphinx
       ${CMAKE_CURRENT_BINARY_DIR}/sphinx-html
-      WORKING_DIRECTORY doc
       COMMENT "Generating sphinx documentation"
       )
   ELSE(MSVC)
     ADD_CUSTOM_TARGET(sphinx-doc
       COMMAND ${SPHINX_BUILD} -b html ${CMAKE_CURRENT_BINARY_DIR}/sphinx
       ${CMAKE_CURRENT_BINARY_DIR}/sphinx-html
-      WORKING_DIRECTORY doc
       COMMENT "Generating sphinx documentation"
       )
 
@@ -60,7 +58,6 @@
     ${CMAKE_BINARY_DIR}/doc/sphinx-html
     COMMAND ${SPHINX_BUILD} -b html  ${CMAKE_CURRENT_BINARY_DIR}/sphinx
       ${CMAKE_CURRENT_BINARY_DIR}/sphinx-html
-    WORKING_DIRECTORY doc
     COMMENT "Generating sphinx documentation"
     )
 
